controllers/hibp.js

```diff
@@ -15,7 +15,9 @@ const log = mozlog("controllers.hibp");
 
 // Get addresses and language from either subscribers or
 // email_addresses fields
+
 function getAddressesAndLanguageForEmail(recipient) {
+  const preFxaSubscriber = recipient.fxa_uid ? true : false;
```

Comment:
I added a commit to fix this logic - i.e., an `fxa_uid` means the user is *NOT* a pre-FXA subscriber.